### PR TITLE
feat: add Readme

### PR DESCRIPTION
**Pull request by ice-orion**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
...
+- Set the App environment
+
+  ```
+  yarn configure:staging
+  ## or
+  yarn configure:production
+  ```
+
+  > [!NOTE]
+  > The commands above take the app env files from the `mobile-app-secrets` folder that should be downloaded beforehand and placed on the same level as the mobile-app project
```
Comment (ice-hades) [inline README.md]: can we provide a link to it?

**Comment by ice-hades** [comment]
maybe to mentions match:read and generate-translation-types ?

**Review by ice-atlas — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (ice-orion) [reply README.md]: `mobile-app-secrets` is a private repo, so i'm not sure what is the best way to do so

**Comment by ice-orion** [comment]
> maybe to mentions match:read and generate-translation-types ?

`match:read` is a part of the deployment process - u don't need it to run the app locally on a simulator (plus u need to have access to mobile-app-secrets repo + match repo to use it, if u want to run the app locally on a real device).

I didn't describe the deployment process here since we do it on ci/cd anyway.

As for `generate-translation-types`, yea will add it

upd: added `Localization rules` part

File: README.md (hunk 1 above)
Comment (ice-orion) [reply README.md]: actually, added the link

**Review by ice-hades — APPROVED** [review]

**Review by ice-atlas — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

**Pull request merged**